src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -1,31 +1,43 @@
 import { useIntl } from 'react-intl';
 import { useRouter } from 'next/router';
-import { Box, makeStyles, Tab, Tabs, Typography } from '@material-ui/core';
-import { FunctionComponent, ReactElement } from 'react';
+import { ArrowDownward, ArrowUpward } from '@material-ui/icons';
+import { Box, Button, Collapse, makeStyles, Tab, Tabs, Theme, Typography } from '@material-ui/core';
+import { FunctionComponent, ReactElement, useState } from 'react';
 
 import BreadcrumbTrail from '../../BreadcrumbTrail';
 import OrganizeSidebar from 'components/organize/OrganizeSidebar';
 
-const useStyles = makeStyles((theme) => ({
+interface StyleProps {
+    collapsed: boolean;
+}
+
+const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
     root: {
         [theme.breakpoints.down('xs')]: {
             paddingTop: '3.5rem',
         },
     },
     title: {
         marginBottom: '8px',
+        transition: 'all 0.3s ease',
     },
     titleGrid: {
+        '& .hide-button': {
+            gridColumnEnd: 'none',
+        },
         alignItems: 'center',
         display: 'grid',
         gap: '1rem',
-        gridTemplateColumns:'1fr auto',
+        gridTemplateColumns: () => '1fr auto',
```

Comment:
I wonder why that's needed

Reply:
It wasn't 😅

Reply:
@lowlandjuju Does that mean we should take it out?

Reply:
yes - I thought I had but got distracted